api/src/backend/tasks/jobs/backfill.py

```diff
@@ -341,3 +342,78 @@ def backfill_scan_category_summaries(tenant_id: str, scan_id: str):
         )
 
     return {"status": "backfilled", "categories_count": len(category_counts)}
+
+
+def backfill_scan_resource_group_summaries(tenant_id: str, scan_id: str):
```

Comment:
Why do we need this backfill? I thought this feature will populate data once released as the new `ResourceGroup` was not present before.